### PR TITLE
fix(status-messages): Only report insufficient crew on player-owned ships

### PR DESCRIPTION
**Pull request by petervdmeer**
**Bugfix:** This PR addresses issue #8663

## Fix Details
fixes: 8663

## Testing Done
Tested with the savegame from 8663; didn't get the warning messages.
Also did some capturing (getting my crew below the required crew); and got the relevant warning messages.

## Save File
See #8663 for a savegame to verify that the messages no longer get generated.

### CODE REVIEW

File: source/Ship.cpp [hunk 1]
```diff
@@ -1927,12 +1927,15 @@ void Ship::Move(vector<Visual> &visuals, list<shared_ptr<Flotsam>> &flotsam)
 	else if(requiredCrew && static_cast<int>(Random::Int(requiredCrew)) >= Crew())
 	{
 		pilotError = 30;
-		if(parent.lock() || !isYours)
-			Messages::Add("The " + name + " is moving erratically because there are not enough crew to pilot it."
-				, Messages::Importance::Low);
-		else
-			Messages::Add("Your ship is moving erratically because you do not have enough crew to pilot it."
-				, Messages::Importance::Low);
+		if(isYours)
```
Comment (TomGoodIdea) [inline source/Ship.cpp:1930]:
```suggestion
		if(isYours || (personality.IsEscort() && Preferences::Has("Extra fleet status messages")))
```

**Review by TomGoodIdea — COMMENTED** [review]
Currently, (if I understand the code correctly) it sends the message for **any** non-disabled ships (even bounties in other systems) that don't have enough crew, and that's certainly not good.
But I think knowing that the ship you're escorting doesn't have enough crew can be helpful, for example if you need to wait for them, you know that you'll need to wait longer.

File: source/Ship.cpp (hunk 1 above)
Comment (Zitchas) [reply source/Ship.cpp:1930]: This seems like a good idea. We definitely don't want to know about ships that aren't in our fleet, but ships that *are* in our fleet or among our escorts seems relevant and useful to know.

**Review by TomGoodIdea — APPROVED** [review]

**Comment by petervdmeer** [comment]
Thanks for your review

**Pull request merged**